tests/test.py

```diff
@@ -1,9 +1,9 @@
 import requests
 import time
 import pytest
+import os
 
-# DRS_FILER_URL = os.getenv('DRS_FILER_URL')
-DRS_FILER_URL = "http://localhost:8080/ga4gh/drs/v1"
+DRS_FILER_URL = os.getenv('DRS_FILER_URL')

```

Comment:
I think it would be good to set a default if `DRS_FILER_URL` is not defined:

```py
DRS_FILER_URL = os.getenv('DRS_FILER_URL', 'http://localhost:8080/ga4gh/drs/v1')
```